### PR TITLE
[gdal] Update to 2.4.4

### PR DESCRIPTION
**Pull request by longhuan2018**
update from 2.4.1 to 2.4.4
add new feature latest to support Build latest gdal version (3.0.4)
add new feature kmz to support kmz format by libkml

### CODE REVIEW

File: ports/gdal/CONTROL [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Source: gdal
-Version: 2.4.1-9
+Version: 2.4.4
 Homepage: https://gdal.org/
 Description: The Geographic Data Abstraction Library for reading and writing geospatial raster and vector data.
 Build-Depends: proj4, libpng, geos, sqlite3, curl, expat, libpq, openjpeg, libwebp, libxml2, liblzma, netcdf-c, hdf5, zlib
+Default-Features: latest, kmz
```
Comment (JackBoosY) [inline ports/gdal/CONTROL]: I don't think we should use the latest as default feature.

File: ports/gdal/portfile.cmake [hunk 2]
```diff
@@ -1,12 +1,18 @@
 # vcpkg portfile.cmake for GDAL
 #
 # NOTE: update the version and checksum for new GDAL release
-include(vcpkg_common_functions)
 
-set(GDAL_VERSION_STR "2.4.1")
-set(GDAL_VERSION_PKG "241")
-set(GDAL_VERSION_LIB "204")
-set(GDAL_PACKAGE_SUM "edb9679ee6788334cf18971c803615ac9b1c72bc0c96af8fd4852cb7e8f58e9c4f3d9cb66406bc8654419612e1a7e9d0e62f361712215f4a50120f646bb0a738")
+if (NOT "latest" IN_LIST FEATURES)
```
Comment (JackBoosY) [inline ports/gdal/portfile.cmake]: Only update the version just be okay.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/gdal/CONTROL (hunk 1 above)
Comment (longhuan2018) [reply ports/gdal/CONTROL]: I just use it to test whether the new feature is normal. When it passes, I will delete the default feature.

**Comment by JackBoosY** [comment]
Any updates?

**Comment by longhuan2018** [comment]
> Any updates?

I failed the test under osx, and I found a new PR#9100, which is modifying some compilation methods under linux and osx, so I want to wait for this PR to pass and then do a version update

**Comment by JackBoosY** [comment]
Okay, please ping me when this PR ready for review.

**Comment by JackBoosY** [comment]
Closing this PR since it seems that no progress is being made. Please reopen if work is still being done.

**Pull request closed without merging**